### DISCUSSION TITLE
Adaptive Bed Mesh when using a PRINT_START macro

### DISCUSSION THREAD

**Discussion by Nukle0n** [OrcaSlicer/OrcaSlicer#4943] [Q&A]
As has been recommended many a time, I use a PRINT_START macro in Klipper instead of having all the start Gcode written in the slicer. I don't really understand what I have to do to make Orca do the adaptive meshing, I can make Klipper do the adaptive meshing but if I try the method in the wiki I get either an error saying that the variables aren't defined, or it just does a complete full mesh of the bed.

**Comment by darco415** [comment]
this is what I use w/OS  and Kllipper
print_start BED_TEMP=[bed_temperature_initial_layer_single] EXTRUDER_TEMP=[nozzle_temperature_initial_layer]
